.gitignore

```diff
@@ -1 +1 @@
-node_modules/
\ No newline at end of file
+**/node_modules
```

Comment:
- [ ] Consider adding `node_modules` in .gitignore and fix the path for it like this `node_modules` and consider making sure to remove them from the remote repo.